src/components/Authentication/user.css

```diff
@@ -0,0 +1,56 @@
+:root {
+  --green: #96bf01;
+}
+
+.login-section {
+  height: 500px;
+  background: var(--green);
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
[OPTIONAL] You made some really great design choices here. I have one suggestion though. Consider removing the white space below the login section as the lack of uniformity is not very appealing. You could set the height of the login-section to `100vh` so that it covers the entire page.